__tests__/app.test.js

```diff
@@ -40,6 +40,65 @@ describe("GET", ()=>{
                 })
         })
     })
+    describe("/api/reviews", () => {
+        it("should return status: 200, and return a list of all reviews with specific properties", () => {
+            return request(app)
+                .get("/api/reviews")
+                .expect(200)
+                .then(({body}) => {
+                    body.reviews.forEach((review) => {
+                        expect(review).toEqual(
+                            expect.objectContaining({
+                                review_id: expect.any(Number),
+                                title: expect.any(String),
+                                designer: expect.any(String),
+                                owner: expect.any(String),
+                                review_img_url: expect.any(String),
+                                review_body: expect.any(String),
+                                category: expect.any(String),
+                                created_at: expect.any(String),
+                                votes: expect.any(Number),
+                                comment_count: expect.any(Number)
+                            })
+                        )
+                    })
+                })
+        })
+    })
+    describe("/api/reviews?category", () => {
+        it("should return status: 200, and return a list of all reviews filtered by the given query category", () => {
+            return request(app)
+                .get("/api/reviews?category=social deduction")
+                .expect(200)
+                .then(({body}) => {
+                    expect(body.reviews.length).toBeGreaterThan(0)
+                    body.reviews.forEach((review) => {
+                        expect(review).toEqual(
+                            expect.objectContaining({
+                                review_id: expect.any(Number),
+                                title: expect.any(String),
+                                designer: expect.any(String),
+                                owner: expect.any(String),
+                                review_img_url: expect.any(String),
+                                review_body: expect.any(String),
+                                category: 'social deduction',
+                                created_at: expect.any(String),
+                                votes: expect.any(Number),
+                                comment_count: expect.any(Number)
+                            })
+                        )
+                    })
+                })
+        })
+        it("should return status: 200, and return an empty array when a category has no reviews yet", () => {
+            return request(app)
+                .get("/api/reviews?category=children's games")
+                .expect(200)
+                .then(({body}) => {
+                    expect(body.reviews).toEqual([])
+                    })
+        })
+    })
```

Comment:
I would like to see one more test for if the category does not yet exist in the database.

Reply:
I've included this test further down on line 274 in my error handling test block. Would it be better to reorganise my tests so they are next to the function that they are error handling for?